floris/simulation/floris.py

```diff
@@ -253,6 +254,42 @@ def solve_for_viz(self):
         else:
             full_flow_sequential_solver(self.farm, self.flow_field, self.grid, self.wake)
 
+    def solve_for_points(self, points_x, points_y, points_z):
+        # Do the calculation with the TurbineGrid for a single wind speed
+        # and wind direction and 1 point on the grid. Then, use the result
+        # to construct the full flow field grid.
+        # This function call should be for a single wind direction and wind speed
+        # since the memory consumption is very large.
+
+        vel_model = self.wake.model_strings["velocity_model"]
+
+        if vel_model=="cc":
```

Comment:
Is it correct to have these if-statements? I'm not sure its relevant for this function.

Reply:
You're right; fixing this now.